### DISCUSSION TITLE
Fighting memory leaks with Bindings.StopTracking()

### DISCUSSION THREAD

**Discussion by abenadar** [microsoft/microsoft-ui-xaml#9040] [Q&A] [answered]
Since my Windows app is suffering from memory leaks from a still unknown reason I am trying different strategies. 
I take that usage of {x:Bind} can be a possible cause, which could be avoided with usage of  Bindings.StopTracking() (mentioned [here](https://github.com/microsoft/microsoft-ui-xaml/issues/6435)). It is also mentioned in one or two Stack Overflow issues.

But where do I find even this method or it's documentation? There is only one mention in the Microsoft drocs of it I could find. ([here](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension)). And this one doesn't contain a link.

What am I missing here? Any help is welcome.

PS: I am using .NET 6 with the Windows App SDK 1.4.

**Comment by DarranRowe** [comment, marked as answer]
This is part of the generated code. You can find it in XamlBindingInfo.Xaml.g.h/XamlBindingInfo.Xaml.g.hpp on the C++ side. I think it should be XamlBindingInfo.Xaml.i.g.cs for C#.
Since it is part of the generated code, it should be treated as undocumented/implementation details. So if you can get something to work then fine, but it is up to you to support it.

**Reply by abenadar** [reply]
Understood, thanks.
